R/cleanup.R

```diff
@@ -693,35 +693,35 @@ cleanup_GeneDesc <- function(prot, column) {
 pick_longer_duplicate <- function(prot, column) {
     col <- sym(column)
 
-    prot$row.orig <- 1:nrow(prot)
+    prot$.data$row.orig <- 1:nrow(prot)
 
     # Get list of duplicates
     dups <- prot %>%
-        group_by(AccNum) %>%
+        group_by(.data$AccNum) %>%
         summarize("count" = n()) %>%
         filter(count > 1) %>%
         arrange(-count) %>%
         merge(prot, by = "AccNum")
 
-    dup_acc <- dups$AccNum
+    dup_acc <- dups$.data$AccNum
 
     longest_rows <- c()
     remove_rows <- c()
     for (acc in dup_acc) {
-        dup_rows <- dups %>% filter(AccNum == acc)
+        dup_rows <- dups %>% filter(.data$AccNum == acc)
 
         longest <- dup_rows[which(nchar(pull(dup_rows, {{ col }})) == max(nchar(pull(dup_rows, {{ col }}))))[1], "row.orig"]
 
         longest_rows <- c(longest_rows, longest)
 
-        to_remove <- dup_rows[which(dup_rows$row.orig != longest), "row.orig"][]
+        to_remove <- dup_rows[which(dup_rows$.data$row.orig != longest), "row.orig"][]
 
         # dup_rows[which(nchar(pull(dup_rows,{{col}})) == max(nchar(pull(dup_rows,{{col}}))))[2:nrow(dup_rows)], "row.orig"]
         remove_rows <- c(remove_rows, to_remove)
     }
 
     # grab all the longest rows
-    unique_dups <- prot[-remove_rows, ] %>% select(-row.orig)
+    unique_dups <- prot[-remove_rows, ] %>% select(-.data$row.orig)
```

Comment:
This is likely where the warning was generated from for this function and this is the correct application of the .data prefix